Dockerfile

```diff
@@ -7,12 +7,20 @@ WORKDIR /srv/node
 
 RUN buildDeps=' \
     git \
+    yarn \
     ' && \
+    # `apt-transport-https` is required to use https deb repositories
+    apt-get update -y && \
+    apt-get install -y --no-install-recommends apt-transport-https && \
+    # configure Yarn repository, see: https://yarnpkg.com/en/docs/install#linux-tab
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg|apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list && \
+    # the base image installs yarn, let's be sure we use ours
+    rm -f /usr/local/bin/yarn /usr/local/bin/yarnpkg && \
```

Comment:
Is this really necessary? Doesn't `apt-get install yarn` just do what we want?

Reply:
I am not sure how `PATH` is configured, and I dont trust `update-alternatives` to take care of it. That would be safer in my opinion.